.github/workflows/publish-docs.yml

```diff
@@ -1,8 +1,26 @@
 name: "typedoc"
 
 on:
-  push:
-    branches: [docs]
+  schedule:
+    - cron: "0 0 * * 1-5"
+  workflow_dispatch:
+    inputs:
+      live-run:
+        type: boolean
+        description: Live-run
+        required: false
+      version:
+        type: string
+        description: Release number
+        required: false
+      zenoh-version:
+        type: string
+        description: Release number of Zenoh
+        required: false
+      branch:
+        type: string
+        description: Release branch
+        required: false
 
```

Comment:
It doesn't look like the inputs are used in the workflow so I think you can remove all of them. Just leave the schedule and the empty `workflow_dispatch` so the workflow can be called manually.

Reply:
Makes sense, 
Ive added push on tags, so that docs will automatically get published when a new version is released.